### DISCUSSION TITLE
Recovery Image Download

### DISCUSSION THREAD

**Discussion by ikolsteren** [pgale/lineage_15.1_Installation_TSV#34] [Q&A] [answered]
Maybe its me but that link is not working for me
https://s3.us-west-1.wasabisys.com/rom-release/LineageOS/15.1/starfire/lineage-15.1-20240531-UNOFFICIAL-starfire-recovery.img

**Comment by pgale** [comment, marked as answer]
Still working here in the UK. Must be your setup or ISP. Maybe try a VPN?

**Reply by ikolsteren** [reply]
Thank you - very odd but got it - not sure why the other ones worked no issue

**Reply by justynbell** [reply]
@pgale Not sure if I'm having the same issue as @ikolsteren, but I also can't download the recovery image.  I've tried a couple different browsers and same issue. I'm in the US.

Could you version control them in this repo?  Or is there any other way I can grab it?  I've spent the past few nights trying to set up the tablet in a few different ways, and I'm hitting roadblocks at every turn.  It seems like that link is literally the only link to the recovery image that I can find via google.
